index.html

```diff
@@ -968,17 +951,22 @@ <h4><code>Timing-Allow-Origin</code> Response Header</h4>
 whether a resource's timing information can be shared with the
 <a>current document</a>, is as follows:</p>
 <ol>
-<li>
-<p>If the resource is same origin, return <code>pass</code>.</p>
-</li>
-<li>
-<p>If the <a>Timing-Allow-Origin</a> header value list contains a
-case-sensitive match for the value of the <code><a data-cite=
-"RFC6454#section-4" data-lt="http-origin">origin</a></code> of the
-<a>current document</a>, or a wildcard ("<code>*</code>"), return
-<code>pass</code>.</p>
-</li>
-<li>Return <code>fail</code>.</li>
+    <li><p>Let <var>tainted</var> be false.</p></li>
+    <li><p>For each request in the resource <a
+    data-cite="FETCH#concept-fetch">fetch</a> redirection-chain:</p>
+        <ol>
+            <li><p>If the request is <a>cross-origin</a>, set
```

Comment:
Cross-origin with respect to the original frame, yea?

Reply:
indeed. I'll clarify

Reply:
Yeah, clarified now